implementations/elixir/ockam/ockam_cloud_node/lib/cloud_node/cleanup.ex

```diff
@@ -26,8 +26,10 @@ defmodule Ockam.CloudNode.Cleanup do
 
     case cleanup_kafka_topics do
       true ->
-        Logger.info("Cleanup kafka topics older than #{idle_timeout} ms")
-        TopicCleanup.cleanup_idle_topics(idle_timeout, [])
+        if Code.ensure_loaded?(Ockam.Kafka) do
```

Comment:
not for now maybe we could put things like this in some API  like  `Plugin.enabled?(Kafka)`  or something similar?